Docs/developer_guide/script_repository/sequences.md

```diff
@@ -1,192 +1,192 @@
-## Sequences

```

Comment:
Since some of the markdown file currently have `CRLF` endings, we should bulk fix them. Until this is done, we should avoid updating the entire file.

Reply:
See https://github.com/Slicer/Slicer/pull/6734